Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/openshift/cluster-autoscaler-operator
 COPY . .
 ENV NO_DOCKER=1
 ENV BUILD_DEST=/go/bin/cluster-autoscaler-operator
-RUN unset VERSION && make build
+RUN unset VERSION && GOPROXY=off make build
```

Comment:
@spangenberg why do we need to set `GOPROXY=off` explicitly?

Reply:
Also wouldn't `build` need to be aware of `GO111MODULE` and `GOFLAGS`

Reply:
The `Makefile` is setting `GO111MODULE=on` and `GOFLAGS=...`.
`GOPROXY=off` is done here to really make sure it's not fetching anything else and only uses vendored packages in order to guarantee reproducible builds that are hermetic.

Reply:
thanks! so is `GOPROXY` used by go build? my understanding was that it will just look at the vendored folder by setting `GO111MODULE` and -mod=vendor

Reply:
It does, but in case there is something not vendored yet. Because someone forgot to vendor it. Go build would fetch the missing one live from remote.
This way we fail fast.